### ISSUE TITLE
./setup: ligne 22: dev : command not found

### ISSUE THREAD

**Issue by julienloizelet** [JeroenBoersma/docker-compose-development#106]
I am working on Ubuntu 18.04 and when I launch the command `bin/dev setup`, I get the following error : `./setup: ligne 22: dev : command not found`

The error disappears if I add the line 
`export PATH=/path-to-my-local-repo/bin:${PATH};` in the `.bash_aliases` file.

**Comment by royarthur-dev** [comment]
Hi Julien,

Did you tried to add this line : `export PATH=/home/username/development/bin:${PATH};` at the end of your `.bashrc` file ?

In my case, the working directory is located in : `/home/jerome/development/workspace'.

**Comment by julienloizelet** [comment]
> Hi Julien,
> 
> Have you tried to add this line : `export PATH=/home/username/development/bin:${PATH};` at the end of your `.bashrc` file ?
> 
> In my case, the working directory is located in : `/home/username/development/workspace'.

Hi, yes I did and it worked. But I guess it is supposed to work without adding this line (as in the README, this add is marked as optional).

And I would prefer to not add this line as I see that it will for example imply that the `composer` program will be always launch from the `/home/username/development/bin/composer` (this is what return a `which composer`). I would prefer to keep composer where I installed it manually for other kind of development.
